src/internal/OrdinoApp.tsx

```diff
@@ -420,7 +422,9 @@ export class OrdinoApp extends React.Component<IOrdinoAppProps, IOrdinoAppState>
     view.off(ViewWrapper.EVENT_CHOOSE_NEXT_VIEW, this.chooseNextView);
     view.off(ViewWrapper.EVENT_REPLACE_VIEW, this.replaceViewInViewWrapper);
     view.off(AView.EVENT_ITEM_SELECT, this.updateSelection);
-
+    view.off(AView.EVENT_UPDATE_ENTRY_POINT, (namedSet) => {
+      GlobalEventHandler.getInstance().fire(AView.EVENT_UPDATE_ENTRY_POINT, namedSet);
+    });
```

Comment:
This does not work, because you must remove the event listener with the same function reference (see the listeners above). Using a new function here, won't remove the initial function you registered above.

Reply:
Done